src/Commands/DebugMode.php

```diff
@@ -42,29 +44,37 @@ public function execute(InputInterface $input, OutputInterface $output)
         $debugMode = new DebugAdapter();
         $isDebugModEnabled = $debugMode->isDebugModeEnabled();
 
-        //check if action is allowed
-        if ( !in_array($action,$this->allowed_command_states)){
+        if (!in_array($action, self::ALLOWED_COMMAND_STATES)) {
             $io->error('Action not allowed');
             return false;
         }
 
-        //Define Toggle action
+        //Status
+        if ($action == 'status') {
+            $io->note('Current debug mode : ' . ((true === $isDebugModEnabled) ? 'enabled' : 'disabled'));
```

Comment:
- To display status, I've used text() instead of note().
- For the message, maybe, just Debug mode is on
We have to harmonize this.

Reply:
Sure we need to define which io function use to display for each cases : 
- error
- success
- info

On my previous console tools i was used to use the writeln function with a <info> tag
![image](https://user-images.githubusercontent.com/7163132/69762886-e044ce80-116b-11ea-8756-65dbbee2df01.png)

The same with the wording.
We are just in the beginning of something so it's not really efficient for now :D

Reply:
We should see how doctrine or other SF famous command lines do